rotkehlchen/api/rest.py

```diff
@@ -4196,7 +4196,7 @@ def get_ens_avatar(self, ens_name: str, match_header: Optional[str]) -> Response
 
         Also supports etag mechanism that helps with caching on the client side.
         """
-        avatars_dir = self.rotkehlchen.data_dir / 'icons' / 'avatars'
+        avatars_dir = self.rotkehlchen.data_dir / 'images' / 'avatars'
```

Comment:
What are you doing? Why are you renaming this? Undo. The `icons` directory already exists in all the user's systems. You are not going to create another one. If we ever wanted to do changes there we would need a migration.